tests/test_vercel_ai.py

```
@@ -8749,6 +8749,91 @@ async def event_generator():
+                ToolReturnPart(
+                    tool_name='my_tool',
+                    content='The tool call was interrupted before a result was produced.',
+                    tool_call_id='tc_int',
+                    outcome='interrupted',
+                ),
+            ]
+        ),
+    ]
+
+    ui_messages = VercelAIAdapter.dump_messages(messages)
+    assistant_parts = [msg.model_dump() for msg in ui_messages if msg.role == 'assistant'][0]['parts']
```

Ruff (0.15.20)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)